packages/aft/pubspec.yaml

```diff
@@ -8,14 +8,26 @@ environment:
 
 dependencies: 
   args: ^2.3.0
-  async: ^2.9.0
+  async: ^2.8.0
   aws_common: ^0.1.0
   checked_yaml: ^2.0.0
   cli_util: ^0.3.5
   collection: ^1.16.0
+  flutter_tools:
+    git:
+      url: https://github.com/flutter/flutter.git
+      ref: master
```

Comment:
Should we pin to a commit?

Reply:
For flutter-tools maybe, but I'd think for pub there might potentially be more chance of security updates that we'd want to get automatically?